modules/js-and-css/partytown-web-worker/load.php

```diff
@@ -0,0 +1,139 @@
+<?php
+/**
+ * Module Name: Partytown Web Worker
```

Comment:
Do we really want to call the module Partytown?

IMHO usage of the Partytown library is just an implementation detail.

Reply:
Agreed. A less-specific name would be "Web Worker Offloading". I asked Bard for some alternative names, and it suggested "Threadshift". I like that, but it is trademarked.

Reply:
That makes sense. If you have a certain name in mind, do let me know, and I will edit it.

Reply:
How about Web Worker Offloading for now?

Reply:
SGTM, some more names:
- Scripts offloading/offloader (in the description add a reference to Web Worker and/or Partytown).
- Third-party JS offloader
- Non-critical JS offloader

I will defer name selection to you.

Reply:
Those are some good options but I like Web Worker Offloading.

The JS being offloaded may not be third-party. It may also be critical, but just offloaded so it can run in another thread.

Reply:
```suggestion
 * Module Name: Web Worker Offloading
```